src/pages/blog/[...page].astro

```diff
@@ -12,23 +12,34 @@ const { page } = Astro.props;
 ---
 
 <BaseLayout title="Blog">
-  <ul>
-    {
-      page.data.map((post) => (
-        <>
-          <HorizontalCard
-            title={post.data.title}
-            img={post.data.heroImage}
-            desc={post.data.description}
-            url={"/blog/" + post.slug}
-            target="_self"
-            badge={post.data.badge}
-          />
-          <div class="divider my-0" />
-        </>
-      ))
-    }
-  </ul>
+  <div class="mb-5">
+      <div class="text-3xl w-full font-bold">Blog</div>
+    </div>
+  
+  {       
+    page.data.length === 0 ? (
+      <div class="bg-orange-100 border-l-4 border-orange-500 text-orange-700 w-full p-4" role="alert">
```

Comment:
This template use different themes and in some themes the orange is too strident color 

![Screenshot from 2023-04-18 14-27-06](https://user-images.githubusercontent.com/53962116/232887187-5ae7c597-2c49-4a70-abdb-468eb5a26647.png)


We can use something like this using the DaisyUI color scheme

```  
<div class="bg-base-200 border-l-4 border-secondary w-full p-4 min-w-full">
        <p class="font-bold">Sorry!</p>
        <p>There are no blog posts to show at the moment. Check back later!</p>
      </div>
```